internal/storagetools/put_object_handler.go

```diff
@@ -0,0 +1,51 @@
+package storagetools
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/wal-g/wal-g/utility"
+
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/ioextensions"
+)
+
+func HandlePutObject(localPath, dstPath string, uploader *internal.Uploader, overwrite bool) {
```

Comment:
do we need compression and encryption here ?

Reply:
Initially, I thought that we can control it via the config settings. But I agree that it would be more flexible to add the `--no-compress` and `--no-encrypt `flags here.

Reply:
I implemented the `--no-compres` and `--no-encrypt` flags in 02ad732